development/gcp/pkg/cloudfunctions/logging.go

```diff
@@ -60,18 +59,24 @@ func (e *LogEntry) WithComponent(component string) *LogEntry {
 func (e LogEntry) LogCritical(message string) {
 	e.Severity = "CRITICAL"
 	e.Message = message
-	log.Println(e)
+	fmt.Println(e)
 	panic(message)
 }
```

Comment:
Just a question about `message` arg. Wouldn't it be more convenient to use `fmt.Sprintf`, as it will give even better readability of the code where you use this function. 
```go
func (e LogEntry) LogCritical(format string, args ...string) {
    message := fmt.Sprintf(format, args...)
    e.Severity = "CRITICAL"
    e.Message = message
    fmt.Println(e)
    panic(message)
}
```